wcomponents-theme/src/main/js/wc/ui/table/action.js

```diff
@@ -39,17 +39,21 @@ define(["wc/dom/event",
 					table,
 					// todo: this filter can be deleted once we drop WDataTable as rows will no longer be able to be disabled
 					filter = getFilteredGroup.FILTERS.enabled | getFilteredGroup.FILTERS.selected,
+					otherSelected = 0,
 					selected = 0;
 
 				if ((table = ACTION_TABLE.findAncestor(button))) {
 					min = condition.min;
 					max = condition.max;
 
+					otherSelected = condition.otherSelected;
+
 					if (!(min || max)) { // no condition worth testing.
 						return true;
 					}
 
-					selected = (getFilteredGroup(ROW_CONTAINER.findDescendant(table), {filter: filter})).length;
+					selected = parseInt((getFilteredGroup(ROW_CONTAINER.findDescendant(table), {filter: filter})).length)+
```

Comment:
Can you explain your use of parseInt here?

Reply:
I needed to add the integers together (currently selected + other selected) for the total, and if I just did `selected = (getFilteredGroup(ROW_CONTAINER.findDescendant(table), {filter: filter})).length + other` it would return `NaN`. I then tried to parseInt the entire thing but then they would just concatenate like a string, so I had to parseInt both to actually get them to add together.

Reply:
Given that the length of an Array is an integer if the object is, in fact, an Array what else could be causing the left side of a `+` operator to cause a concatenation? Hint: `parseInt("77trombones") === 77` whereas `parseInt("trombones77") === NaN`. How would you test both operands to ensure they are both typed as expected?